app/views/devise/registrations/edit_password.html.erb

```diff
@@ -18,37 +18,24 @@
     <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do %>
     <%= render "devise/shared/error_messages", resource: resource %>
 
-      <%= render "govuk_publishing_components/components/input", {
+      <%= render "govuk_publishing_components/components/show_password", {
         label: {
           text: t("devise.registrations.edit.fields.password.label"),
         },
         hint: t("devise.registrations.edit.fields.password.hint"),
         name: "user[password]",
-        type: "password",
         id: "password",
         error_message: devise_error_items(:password),
         autocomplete: "new-password",
       } %>
 
-      <%= render "govuk_publishing_components/components/input", {
-        label: {
-          text: t("devise.registrations.edit.fields.password_confirm.label"),
-        },
-        name: "user[password_confirmation]",
-        type: "password",
-        id: "password_confirmation",
-        error_message: devise_error_items(:password_confirmation),
-        autocomplete: "new-password",
-      } %>
-
-      <%= render "govuk_publishing_components/components/input", {
+      <%= render "govuk_publishing_components/components/show_password", {
```

Comment:
For what it's worth, I went through this twice and both times had a _d'oh_ moment by putting a new password in twice and got the current password is wrong error